apps/pie-storybook/stories/pie-card-container.stories.ts

```diff
@@ -98,7 +98,10 @@ const Template = ({
     slot,
     aria,
     variant,
-}: CardContainerProps): TemplateResult => html`
+}: CardContainerProps): TemplateResult => {
+    const darkMode = ['inverse', 'outline-inverse'].includes(variant);
```

Comment:
Your way is more precise but if we did this it would capture any future inverse variants
```suggestion
    const darkMode = variant.includes('inverse');
```

Reply:
good one!